### PR TITLE
Add Hints to simultaneous configuring in two screens

### PR DESCRIPTION
**Pull request by rubo77**
follow-up of #46

### CODE REVIEW

**Comment by RalfJung** [comment]
I have merged the parts I plan to merge in #46, what is left to do here?

I told you I don't think the README should recommend working in two shells at the same time.

**Comment by rubo77** [comment]
I thought, the discussion should take place here.

I didn't get the point why it would be too complicated to work in two screens

File: README.md [hunk 1]
```diff
@@ -52,31 +53,19 @@ Make sure, you have at least 1.5 GB of free diskspace.
     ```
 
 
-5.  If you want to save some disk space, you can now clean up the build directory:
+    > The build process takes a while, you can continue with steps 5. to 8. in a
+    > second shell as the build process continues.
```
Comment (RalfJung) [inline README.md]: I'd be fine with adding just a remark like this. But I don't like how you are restructuring the entire README here for the purpose of parallel processing. That's just optimizing for the wrong case.

**Comment by rubo77** [comment]
like this?

**Comment by RalfJung** [comment]
Yes, thanks!

**Pull request merged**